deploy-airflow-dags/README.md

```diff
@@ -0,0 +1,27 @@
+# deploy-airflow-dags
+This action allows to deploy dags to Airflow instance.
```

Comment:
suggestion: describe what action really do, it expects to be launched from a repo that have some preconditions (i.e presence of a "dags" dir , shared volume with airflow instance...etc)
One can think looking at the parameters that this is self contained (deploy dags using airflow API somehow), instead it hardly depends on the environment it's launched on